pymagicc/__init__.py

```diff
@@ -10,28 +10,25 @@
 
 # -*- coding: utf-8 -*-
 
-from .compat import get_param
-from .paths import _get_magicc_paths
-from .run import MAGICC
-
 import datetime
 import linecache
 import logging
 import os
-import platform
 import subprocess
 
 import f90nml
 import pandas as pd
 
 from ._version import get_versions
+from .config import config as _config
+from .run import MAGICC6, MAGICC7  # noqa
```

Comment:
what does `noqa` mean?

Reply:
No "quality assurance" -- tell e.g. `flake8` not to worry about this line

Reply:
It tells the linter to ignore this line. flake8 doesn't see any code which uses `MAGICC7` so it raises an error. MAGICC7 was imported for use by package users `from pymagicc import MAGICC7`